packages/esm-patient-attachments-app/src/camera-media-uploader/capture-photo.component.tsx

```diff
@@ -29,12 +32,18 @@ const CapturePhoto: React.FC<CapturePhotoProps> = ({ initialState, onCapturePhot
   }, [onCapturePhoto]);
 
   return (
-    <div style={{ display: 'flex', alignItems: 'center' }}>
-      <div style={{ maxWidth: '64px' }}>
-        <img src={dataUri || initialState || placeholder} alt="Preview" style={{ width: '100%' }} />
+    <div className={styles.container}>
+      <div className={styles.imageContainer}>
+        {dataUri || initialState ? <img src={dataUri || initialState} alt="Preview" /> : <p>No image to display</p>}
       </div>
-      <Button kind="ghost" onClick={showCam} style={{ flex: 1 }}>
-        {initialState ? t('changeImage', 'Change image') : t('addImage', 'Add image +')}
+      <Button
+        kind="secondary"
+        onClick={showCam}
+        className={styles.button}
+        renderIcon={Edit}
+        size={isTablet ? 'lg' : 'sm'}
+      >
+        {initialState ? t('changeImage', 'Change') : t('editImage', 'Edit')}
```

Comment:
```suggestion
        {(initialState || dataUri) ? t('changeImage', 'Change') : t('editImage', 'Edit')}
```

Reply:
Better is to make a variable 
`const isImageAvailable = dataUri ?? initialState`